src/federation/inbox.ts

```diff
@@ -173,6 +174,18 @@ export async function onFollowAccepted(
     inboxLogger.debug("Invalid actor: {actor}", { actor });
     return;
   }
+
+  const isRelayAccept = accept.objectId?.hash.includes("#relay-follows/");
+
+  if (isRelayAccept) {
+    // Update relay state to accepted if the follow is a relay
+    await db
+      .update(relays)
+      .set({ state: "accepted" })
+      .where(eq(relays.followRequestId, accept.objectId!.href));
+    return;
```

Comment:
It would be better if it's extracted into another function like `onRelayAccepted()`, and the above conditional is moved to *src/federation/index.ts*.  The following is an example how to branch cases:

https://github.com/fedify-dev/hollo/blob/f985afe1cb70e499498a7cf90d3c2d7325e3de0b/src/federation/index.ts#L112-L133